packages/mdc-button/_mixins.scss

```diff
@@ -113,6 +113,22 @@
   line-height: $mdc-dense-button-height;
 }
 
+@mixin mdc-button-filled-accessible($container-color) {
```

Comment:
I made the mistake in the FAB PR to not add demos at the same time I added the public mixin. Can you add a demo of using mdc-button-filled-accessible to the demo page in this PR?

Reply:
Good point. Let me confirm with @kfranqueiro if this will conflict with his work for separating `demos.scss`.